src/Card/Card.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Card = ({movie}) => {
+
+  console.log(movie);
+  return (
+    <article>
+      <p className="movie-title">
+        {movie.title}
+      </p>
+      <p className="movie-rating">
+        {movie.average_rating}
+      </p>
+      <p className="movie-date">
+        {movie.release_date}
```

Comment:
We should double check NOW to make sure that the mock data and the API data are structured the same way, even though Leta said they are.